menu/menu.yaml

```diff
@@ -516,6 +516,8 @@
         Title: Version 2.x to 2.2.4
       - Url: transports/upgrades/sqlserver-1.x-1.2.5
         Title: Version 1.x to 1.2.5
+      - Url: transports/upgrades/sqlserver-unicode-headers
```

Comment:
Good catch. I'll move it right below 3.1 upgrade guide to keep it ordered by version numbers.